lib/components/Input/base/index.tsx

```diff
@@ -0,0 +1,149 @@
+import { QuillSvgProps } from "@deriv/quill-icons";
+import clsx from "clsx";
+import {
+    ForwardRefExoticComponent,
+    HTMLInputTypeAttribute,
+    InputHTMLAttributes,
+    forwardRef,
+    useState,
+} from "react";
+import "./base.scss";
+import React from "react";
+import { TMediumSizes } from "../../../types";
+
+export interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
+    type?: HTMLInputTypeAttribute;
+    icon?: ForwardRefExoticComponent<Omit<QuillSvgProps, "ref">>;
+    statusIcon?: ForwardRefExoticComponent<Omit<QuillSvgProps, "ref">>;
+    inputSize?: TMediumSizes;
+    status?: "neutral" | "success" | "error";
+    disabled?: boolean;
+    variant?: "fill" | "outline";
+    leftStatusMessage?: string;
+    rightStatusMessage?: string;
+    textAlignment?: "left" | "center";
+    label?: string;
```

Comment:
Can label be a component? Or only string

Reply:
as per the figma, only string

Reply:
Changed it to react node